### PR TITLE
Cleanup readme and update package.json

### PR DESCRIPTION
**Pull request by Kicu**
## Done:
 - updated `react-native-svg` version in peer deps. I have talked with @WoLewicki who is one of maintainers of https://github.com/software-mansion/react-native-svg and we should be supporting versions 14+
 - updated README
 - provided PR template for future contributors

### CODE REVIEW

**Review by WoLewicki — APPROVED** [review]
LGTM!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+## What this fixes
+<!--
+  Any details about the bug this fixes (if there was any).
+  Related issues (if there were any).
+-->
+
+## Checklist
+- [ ] I have described the bug/issue
+- [ ] I have provided reproduction in `Example` App
+- [ ] I have that solution works on `Example` App on all platforms:
```
Comment (luacmartins) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] I have tested that solution works on `Example` App on all platforms:
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - Android
+  - Ios
```
Comment (luacmartins) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
  - iOS
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  - web
```
Comment (luacmartins) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
  - Web
```

**Review by luacmartins — CHANGES_REQUESTED** [review]
Minor copy changes

**Review by luacmartins — APPROVED** [review]

**Pull request merged**